docs/getting-started/installation.md

```diff
@@ -0,0 +1,51 @@
+# How to install propsd #
+
+[Releases of propsd][releases] include both source tarballs and Debian
+packages. Debian and Ubuntu based Linux distributions can use the pre-built
```

Comment:
I update the "releases" related links so they now go to the latest release instead of the "look, here's every possible release" page. That should make it easier to get started when you're reading and clicking through the guide.